sci-libs/rbio/rbio-4.2.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+Sparse_PV="7.3.1"
+Sparse_P="SuiteSparse-${Sparse_PV}"
+DESCRIPTION="Sparse matrices Rutherford/Boeing format tools"
+HOMEPAGE="https://people.engr.tamu.edu/davis/suitesparse.html"
+SRC_URI="https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/refs/tags/v${Sparse_PV}.tar.gz -> ${Sparse_P}.gh.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0/4"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
```

Comment:
Since this one is new, please add only KEYWORDS you have tested it on.

Reply:
Fair.